mcs/class/System/Mono.UnityTls/UnityTlsProvider.cs

```diff
@@ -114,29 +119,42 @@ unsafe internal class UnityTlsProvider : MonoTlsProvider
 						var trustCAnativeRef = UnityTls.NativeInterface.unitytls_x509list_get_ref (trustCAnative, &errorState);
 
 						fixed (byte* targetHostUtf8Ptr = targetHostUtf8) {
-							result = UnityTls.NativeInterface.unitytls_x509verify_explicit_ca (certificatesNativeRef, trustCAnativeRef, targetHostUtf8Ptr, (size_t)targetHostUtf8.Length, null, null, &errorState);
+							result = UnityTls.NativeInterface.unitytls_x509verify_explicit_ca (
+								certificatesNativeRef, trustCAnativeRef, targetHostUtf8Ptr, (size_t)targetHostUtf8.Length, x509verify_callback, finalCertificateChainNative, &errorState);
```

Comment:
Does something need done to keep the `x509verify_callback` alive? Now long is the function pointer active? The duration of the call or longer?

Reply:
Only the duration of the call

Reply:
Okay, we may still need something to keep this alive, but it's likely not pressing.